### PR TITLE
feat: introduce input component

### LINKED ISSUE #10: Extend UI Kit – Inputfield
We need an Inputfield to further extend the UI Kit.
inputfield

### PR DESCRIPTION
**Pull request by Alexpeschel**
fix #10

### CODE REVIEW

File: patterns/atoms/input/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	border: 1px solid ${colors.grey70.toString()};
+	border-radius: 3px;
+	font-size: 15px;
+	color: ${colors.black.toString()};
+
+	${(props: InputProps) => (props.errorText ? "border-color: red;" : "")};
+
+	:focus {
+		border-color: #499fe4;
+		box-shadow: 0px 0px 2px 0px rgba(73, 159, 228, 1);
```
Comment (LKuechler) [inline patterns/atoms/input/index.tsx]: do we have a variable for that in the color.less?

File: patterns/atoms/input/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+const StyledInput = styled.input`
+	box-sizing: border-box;
+	width: 100%;
+	padding: 12px;
+	border: 1px solid ${colors.grey70.toString()};
+	border-radius: 3px;
+	font-size: 15px;
+	color: ${colors.black.toString()};
+
+	${(props: InputProps) => (props.errorText ? "border-color: red;" : "")};
```
Comment (LKuechler) [inline patterns/atoms/input/index.tsx]: do we have a variable for that in the color.less?

File: patterns/atoms/input/index.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	:focus {
+		border-color: #499fe4;
+		box-shadow: 0px 0px 2px 0px rgba(73, 159, 228, 1);
+		outline: 0;
+	}
+`;
+
+const StyledError = styled.div`
+	padding: 9px 3px 0;
+	color: red;
```
Comment (LKuechler) [inline patterns/atoms/input/index.tsx]: do we have a variable for that in the color.less?

**Comment by LKuechler** [comment]
The last commit message looks a bit strange.
Maybe something like this:
`feat(colors): add "red" and "blue light" color variables`

**Comment by Alexpeschel** [comment]
LabelText needs uppercase property from updated copy branch

**Pull request merged**